src/shared/json.c

```diff
@@ -3351,6 +3351,7 @@ int json_parse_file_at(
 int json_buildv(JsonVariant **ret, va_list ap) {
         JsonStack *stack = NULL;
         size_t n_stack = 1;
+        const char *n = NULL;
```

Comment:
i think it's ok to move this to a more outer scope, but maybe not quite as far, i.e. inside the for loop would suffice too

Reply:
It needs to be in this scope as the value needs to survive more than one iteration of the for loop